packages/react-components/src/components/datatree-output-component.tsx

```diff
@@ -63,6 +63,9 @@ export class DataTreeOutputComponent extends AbstractOutputComponent<AbstractOut
             }
```

Comment:
We need to set the state also when the tspClientResponse is OK but the treeResponse is not (e.g. treeResponse.model is null). It needs an else-clause, see timegraph-output-component.tsx.

Reply:
Good catch! Thanks 🌼